server/src/models/Trip.js

```diff
@@ -0,0 +1,25 @@
+const { ExtendedAPIPlugin } = require("webpack")
```

Comment:
What is this being used for?

Reply:
Well... nothing haha. I don't even know how it got there, probably an autocomplete I didn't notice